www/src/components/homepage/homepage-blog-posts.js

```diff
@@ -114,108 +114,93 @@ const ViewAll = () => (
   </ViewAllStyle>
 )
 
-class HomepageBlogPosts extends Component {
-  desktopMediaQuery
-
-  state = {
-    desktopViewport: false,
-  }
-
-  componentDidMount = () => {
-    this.desktopMediaQuery = window.matchMedia(`(min-width: ${breakpoints[3]}`)
-    this.desktopMediaQuery.addListener(this.updateViewPortState)
-    this.setState({ desktopViewport: this.desktopMediaQuery.matches })
-  }
+const splitPostsToColumns = posts =>
+  posts.reduce(
+    (merge, post, idx) => {
+      if (idx % 2) {
+        merge[1].push(post)
+      } else {
+        merge[0].push(post)
+      }
 
-  componentWillUnmount = () => {
-    this.desktopMediaQuery.removeListener(this.updateViewPortState)
-  }
+      return merge
+    },
+    [[], []]
+  )
+
+export default function HomepageBlogPosts({ posts }) {
+  const [desktopViewport, setDesktopViewport] = useState(false)
+
+  useEffect(() => {
+    const updateViewPortState = () =>
+      setDesktopViewport(desktopMediaQuery.matches)
+    const desktopMediaQuery = window.matchMedia(`(min-width: ${breakpoints[3]}`)
```

Comment:
Ooh, I think we can replace this with `mediaQueries.lg` for clarity.

Reply:
`mediaQueries.lg` returns `@media (min-width: 1000px)`, but we are passing only `(min-width: 1000px)` to window.matchMedia. If we use `mediaQueries.lg` then we will pass extra @media to matchMedia.